mdata/aggmetric_test.go

```diff
@@ -486,14 +486,13 @@ func TestGetAggregated(t *testing.T) {
 	m.Add(30, 30)
 	m.Add(31, 31)
 	m.Add(32, 32)
-	m.Add(40, 40)
```

Comment:
Why remove this?

Reply:
But maybe that was on purpose for testing behavior on the boundary condition? Perhaps your intended test should be an additional case?